Library/ConfigParser.cpp

```diff
@@ -210,10 +210,10 @@ const std::wstring* ConfigParser::GetVariableOriginalName(const std::wstring& st
 }
 
 /*
-** Gets the value of a section variable. Returns true if strValue is set.
-** The selector is stripped from strVariable.
-**
-*/
+ ** Gets the value of a section variable. Returns true if strValue is set.
+ ** The selector is stripped from strVariable.
+ **
+ */

```

Comment:
Please revert these spacing changes. The Rainmeter commenting style is weird (I don't like it either), but lets keep it consistent.